### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by VNTee**
Make front-end of homepage

### CODE REVIEW

File: src/test/java/com/epam/service/UserServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.epam.service;
```
Comment (goponenkoev) [inline src/test/java/com/epam/service/UserServiceImplTest.java]: Why this file like a new in pull request if it already exists in dev?

Comment (VNTee) [reply src/test/java/com/epam/service/UserServiceImplTest.java]: Done

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -1,5 +1,27 @@
 welcome = Welcome to main page
-
+homepage = Homepage
```
Comment (imbirevkotlya) [inline src/main/resources/messages.properties]: i18n is not working on login/registration page

Comment (VNTee) [reply src/main/resources/messages.properties]: Login and registration pages in another tasks

File: src/main/java/com/epam/service/UserDetailsServiceImpl.java [hunk 3]
```diff
@@ -22,7 +23,7 @@
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
         User user = userService.getUser("user");
         Set<GrantedAuthority> roles = new HashSet<>();
-        roles.add(new SimpleGrantedAuthority(user.getRole().getEnumrole()));
+        roles.add(new SimpleGrantedAuthority (UserRoleEnum.USER.getEnumrole()));
```
Comment (goponenkoev) [inline src/main/java/com/epam/service/UserDetailsServiceImpl.java]: Why u hardcode `userRole` here if it should depend on the user from DB?

Comment (VNTee) [reply src/main/java/com/epam/service/UserDetailsServiceImpl.java]: I answered on the same question in another pull request, but this changes additionaly provides me to resolve problem with doubled "getRole" calls.

Comment (goponenkoev) [reply src/main/java/com/epam/service/UserDetailsServiceImpl.java]: Doubled getRole calls...?
But in any case, I think it unnecessary movement, 'cause in our next pull request Lev fixed Security and there it's not hardcode

**Pull request merged**